test/assets/http/An-ComunicaEngine-instance-without-default-source_3074023134/yields-results-for-a-SELECT-query-with-a-string-URL_1694985870/recording.har

```diff
@@ -23,24 +23,19 @@
             {
               "_fromType": "array",
               "name": "user-agent",
-              "value": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"
+              "value": "Comunica/actor-http-node-fetch (Node.js v16.10.0; linux)"
```

Comment:
Might be undesired to have a version- and platform-specific user agent, as this will cause people with Windows and a different Node version to accidentally overwrite these files. I understand why Comunica does it though, so maybe we want to just change the string here.